banzai/tests/e2e-k8s.yaml

```diff
@@ -80,7 +79,7 @@ spec:
         periodSeconds: 1
         timeoutSeconds: 10
     - name: banzai-celery-workers
-      image: @BANZAI_IMAGE@
+      image: banzai:test-latest
```

Comment:
is this change intended? It looks like you're using a locally built image.

Reply:
It is. This is a longer term fix we are making as we move away from jenkins.